R/steps-install.R

```diff
@@ -9,6 +9,9 @@ InstallDeps <- R6Class(
     },
 
     prepare = function() {
+      if (!file.exists("DESCRIPTION")) {
+        stopc("No DESCRIPTION file found. The step_install_deps step and the do_package_checks macro are only available for packages.")
```

Comment:
I recently like to format error messages as

```r
cli_alert_danger("<extended descriptive message>")
stopc("No DESCRIPTION file found.")
```

The advantage is that you can wrap the extended description and also use cli formatting in it. The plain message in the stop call is then just to state the main reason of the error.

Reply:
rlang conditions are better here, see `?rlang::cnd_body` . Here I'm using the pattern you're suggesting, for consistency.

Reply:
Thanks, looks promising.